src/common/templates/_monitoring.tpl

```diff
@@ -43,3 +43,31 @@ prometheus.io/scrape: {{ $enabled | quote}}
   protocol: "TCP"
 {{- end }}
 {{- end -}}
+
+{{/* Podmonitor template to be added for Google Managed prometheus
```

Comment:
so all services/ charts will use this template function for monitoring ? how are we making sure smp customers will not be affected by this change ?